frontend/src/libraries/explorer-wamp/transactions.ts

```diff
@@ -435,6 +454,12 @@ export default class TransactionsApi extends ExplorerApi {
           transactionInfo.hash,
           transactionInfo.signerId,
         ]);
+
+        const transactionReceiptsList = await this.call<any>(
```

Comment:
I do not know whether Frol tell you this but the pattern for this query is actually a little different from current one. We query the list of transactions and outcome for the transactions directly here instead writing on the backend and expose wamp method, please make sure this with Frol to see what is the way he wants

Reply:
@icerove thanks for noticing it. Indeed, it is not following the current pattern, but since we switched from our legacy backend, and have more room on the backend now, I decided we go forward with gradually moving the queries to the backend and eventually forbid the arbitrary `query` requests. Thus, this is our path forward.